src/librbd/WatchNotifyTypes.h

```diff
@@ -73,6 +73,8 @@ enum NotifyOp {
   NOTIFY_OP_SPARSIFY           = 17,
   NOTIFY_OP_QUIESCE            = 18,
   NOTIFY_OP_UNQUIESCE          = 19,
+  NOTIFY_OP_METADATA_SET       = 20,
```

Comment:
Nit: instead of two ops, perhaps combine them into one where the value is wrapped w/ a `std::optional<>` to denote if it's a removal vs set.